test/collins/util/parsers/LshwParserSpec.scala

```diff
@@ -492,5 +492,18 @@ class LshwParserSpec extends mutable.Specification {
       }
     } // wonky opterons
 
+    "Parse GPU server" in {
```

Comment:
I noticed the `vector` vs `list` discrepancy previously, but I _just_ stumbled onto the problem: our CPU frequency is different per processor unlike the existing XML test cases.

I manually edited `<size units="Hz">###</size>` from one of our sample XML files and all tests are now passing from our funky gpu servers.

Interestingly enough, I think this problem would have bubbled up previously if `lshw-b0216.xml` was used as a test case for the `LshwHelperSpec` test (instead of just `LshwParserSpec`).

Thanks for the help, lemme take another look at this.